### ISSUE TITLE
VoiceOver audit & fixes

### ISSUE THREAD

**Issue by kylehickinson** [brave/brave-ios#2841]
### Description: 

There are a few different places in the Brave app that are not properly accessible or could group elements better to read better when a user is using VoiceOver

An example of this is the URL bar, which does not read out entirely correctly. It is noticeable as it is the first element to be focused when the BVC becomes visible in any way

**Comment by Travisnarkle79** [comment]
> ### Description:
> There are a few different places in the Brave app that are not properly accessible or could group elements better to read better when a user is using VoiceOver
> 
> An example of this is the URL bar, which does not read out entirely correctly. It is noticeable as it is the first element to be focused when the BVC becomes visible in any way
